climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1204 @@
+                                      gpw_version=GPW_VERSION,
+                                      data_dir=SYSTEM_DIR, reproject_first=True):
+        """
+        create LitPop exposure for `country` and then crop to given shape.
+
+        Parameters
+        ----------
+        shape : shapely.geometry.Polygon or MultiPolygon or Shape or list of
+            Polygon objects.
+            Geographical shape for which LitPop Exposure is to be initiated.
+        countries : list with str or int
+            list containing country identifiers:
+            iso3alpha (e.g. 'JPN'), iso3num (e.g. 92) or name (e.g. 'Togo')
+        res_arcsec : float (optional)
+            Horizontal resolution in arc-sec.
+            The default is 30 arcsec, this corresponds to roughly 1 km.
+        exponents : tuple of two integers, optional
+            Defining power with which lit (nightlights) and pop (gpw) go into LitPop.
+            Default: (1, 1)
+        fin_mode : str, optional
+            Socio-economic value to be used as an asset base that is disaggregated
+            to the grid points within the country
+            * 'pc': produced capital (Source: World Bank), incl. manufactured or
+                    built assets such as machinery, equipment, and physical structures
+                    (pc is in constant 2014 USD)
+            * 'pop': population count (source: GPW, same as gridded population)
+            * 'gdp': gross-domestic product (Source: World Bank)
+            * 'income_group': gdp multiplied by country's income group+1
+            * 'nfw': non-financial wealth (Source: Credit Suisse, of households only)
+            * 'tw': total wealth (Source: Credit Suisse, of households only)
+            * 'norm': normalized by country
+            * 'none': LitPop per pixel is returned unchanged
+            The default is 'pc'.
+        admin1_calc : boolean, optional
+            If True, distribute admin1-level GDP (if available). Default: False
+        reference_year : int, optional
+            Reference year for data sources. Default: 2020
+        gpw_version : int, optional
+            Version number of GPW population data.
+            The default is GPW_VERSION
+        data_dir : Path, optional
+            redefines path to input data directory. The default is SYSTEM_DIR.
+        reproject_first : boolean, optional
+            First reproject nightlight (Lit) and population (Pop) data to target
+            resolution before combining them as Lit^m * Pop^n?
+            The default is True. Warning: Setting this to False affects the
+            disaggregation results - expert choice only
+
+        Raises
+        ------
+        NotImplementedError
+
+        Returns
+        -------
+        None.
+
+        """
+        shape_list = _shape_to_list(shape)
+        # init country exposure:
+        exp = self._set_one_country(country, res_arcsec=res_arcsec, exponents=exponents,
```

Comment:
As I understand only one country is accepted (not a list of countries as in the description of this function). Why not create `exp = LitPop()` and use the `exp.set_country()` here. Then the function could directly handle multiple countries as in other functions of this class...

Reply:
Thanks for spotting this inconsistency. I have changed it and renamed the method to `set_custom_shape_from_countries`. https://github.com/CLIMADA-project/climada_python/pull/239/commits/72bde19471c96c3d62e85a5261a5ded23c1a8d16, https://github.com/CLIMADA-project/climada_python/pull/239/commits/783df572c078e40fd78b4b462b23d506dd93c307

Reply:
however, I used `self.set_countries` directly instead of `exp = LitPop` .